onnxruntime_extensions/tools/pre_post_processing/steps/vision.py

```diff
@@ -139,6 +139,11 @@ def _create_graph_for_step(self, graph: onnx.GraphProto, onnx_opset: int):
         output_shape_str = f"YCbCr_ppp_{self.step_num}_h, YCbCr_ppp_{self.step_num}_w"
         assert input_type_str == "uint8"
 
+        split_attr = "axis = -1"
+        if onnx_opset >= 18:
+            # Split now requires the number of outputs to be specified even though that can be easily inferred...
+            split_attr += ", num_outputs = 3"
```

Comment:
thanks for the fix!

interesting to know that Split-18 requires num_outputs as an additional attribute now.